artifacts/contracts/interfaces/IFactoryRouter.sol/IFactoryRouter.json

```diff
@@ -104,6 +104,19 @@
       "stateMutability": "nonpayable",
       "type": "function"
     },
+    {
+      "inputs": [],
+      "name": "getConsumeFees",
```

Comment:
Shouldn't it be "fee" not "fees" everywhere?

Reply:
fixed